imageroot/systemd/nextsec-controller-notifier.path

```diff
@@ -0,0 +1,7 @@
+[Unit]
```

Comment:
I'd change also the name of the unit to something like `metrics-exporter-notifier.path`

Reply:
Due to the rootfullness of the module, editing the name of the service/path will make the service not to be picked up [from the configuration process](https://github.com/NethServer/ns8-nextsec-controller/blob/provides-prometheus-metrics/imageroot/actions/create-module/30systemd), we can call it `nextsec-controller-metrics-exporter`, deal?